### ISSUE TITLE
Improve support for region of interest

### ISSUE THREAD

**Issue by gcode-importer** [uclouvain/openjpeg#39]
Originally reported on Google Code with ID 39

```
It would be very useful to be able to specify a rectangular region of interest to be
extracted when using j2k_to_image. This is especially useful when working with very
large images where only small portions of the images are needed.
```

Reported by `keith.hughitt` on 2010-09-15 07:44:48

**Comment by rouault** [comment]
This is addressed by PR #1010

**Comment by rouault** [comment]
Now solved with PR #1010 merged
